cfbs.json

```diff
@@ -883,11 +883,51 @@
       "tags": ["security", "compliance"],
       "repo": "https://github.com/nickanderson/cfengine-lynis",
       "by": "https://github.com/nickanderson",
-      "version": "3.1.1",
-      "commit": "70bdf7be29c890d3bf162a8fe993e8d12eb61388",
+      "version": "3.1.1-2",
+      "commit": "4db6d515c45873a9f3fa7ce78ece593027464b64",
       "steps": [
         "copy policy/main.cf services/lynis/main.cf",
-        "json cfbs/def.json def.json"
+        "json cfbs/def.json def.json",
+        "input ./input.json def.json"
+      ],
+      "input": [
+        {
+          "type": "string",
+          "namespace": "lynis",
+          "bundle": "globals",
+          "variable": "version",
+          "label": "Lynis version",
+          "question": "What version of Lynis should be used?",
+          "default": "3.1.1"
+        },
+        {
+          "type": "string",
+          "namespace": "lynis",
+          "bundle": "globals",
+          "variable": "tar_url",
+          "label": "Tarball url",
+          "question": "Where should clients download lynis from (http/https)",
+          "default": "https://downloads.cisofy.com/lynis/lynis-$(version).tar.gz",
```

Comment:
maybe base-url instead? so that `lynis-$(version).tar.gz` is added in the policy.

Reply:
then again, if they are self-hosting, it would make sense to allow the full URL.